pulsar-client-api/src/main/java/org/apache/pulsar/client/api/SubscriptionIsolationLevel.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.client.api;
+
+public enum SubscriptionIsolationLevel {
```

Comment:
It should be TransactionIsolationLevel

Reply:
I will use the same class mentioned by [PIP-298](https://github.com/apache/pulsar/pull/21246/files#diff-803ac9ece417d293e4d5b24dd080e45f7fdab1f51374f179ccdb38697816eca1R29-R38) to maintain consistency. 

Once this PR is merged, I will comment in the [PR](https://github.com/apache/pulsar/pull/21246) to use the same class.

BTW: Maybe we could also change the Cli flag to: `--sub-isolation-level`, WDYT?

Reply:
I think PIP-289 should also use TransactionIsolationLevel
We don't have any concept of Subscription Isolation.